src/Components/Search/Previews/index.tsx

```diff
@@ -7,16 +8,20 @@ export interface SearchPreviewProps {
   entityType: string
 }
 
+const map = {
```

Comment:
Ugh, I meant to rename this - going to get a commit in that does that real quick.